js/main.js

```diff
@@ -1,141 +1,167 @@
+    { id: 5, name: 'American', roast: 'medium' },
+    { id: 6, name: 'Breakfast', roast: 'medium' },
+    { id: 7, name: 'High', roast: 'dark' },
+    { id: 8, name: 'Continental', roast: 'dark' },
+    { id: 9, name: 'New Orleans', roast: 'dark' },
+    { id: 10, name: 'European', roast: 'dark' },
+    { id: 11, name: 'Espresso', roast: 'dark' },
+    { id: 12, name: 'Viennese', roast: 'dark' },
+    { id: 13, name: 'Italian', roast: 'dark' },
+    { id: 14, name: 'French', roast: 'dark' },
+];
+
+const coffeeContainerElement = document.querySelector('#coffee-container');
+
+const submitButton = document.querySelector('#submit');
+submitButton.addEventListener('click', updateCoffees);
+
+const btnClear = document.querySelector('#btnClear');
+btnClear.addEventListener('click', clearAddForm);
+
+const roastSelection = document.querySelector('#roast-selection');
+roastSelection.addEventListener('change', updateCoffees);
+
+const coffeeName = document.querySelector('#coffeeName');
+coffeeName.addEventListener('input', updateCoffees);
+
+const addForm = document.querySelector("#submitForm");
+addForm.addEventListener('submit', addCoffee);
+
+const addRoastSelection = document.querySelector('#add-roast-selection');
+const addCoffeeName = document.querySelector('#name');
+
+function renderCoffee(coffee) {
+    let htmlOneCofffee = `
     <div class="coffee ${coffee.roast}">
         <h4 class="w-100 text-center">${coffee.name}</h4>
         <p class="w-100 text-center">${coffee.roast}</p>
+        <button class="deleteBtn" type="button" onclick="deleteCoffeeById(${coffee.id})">Delete</button>
     </div>
     `;
-    }
-
-    function renderCoffees(coffees) {
-        let html = '';
-        for (let i = 0; i < coffees.length; i++) {
-            html += renderCoffee(coffees[i]);
-        }
-        return html;
-    }
-
-    function updateCoffees(e) {
-        if (e) {
-            e.preventDefault();
-        }
-        const selectedRoast = roastSelection.value.toLowerCase();
-        const selectedCoffeeName = coffeeName.value.toLowerCase().trim();
-
-        const filteredCoffees = [];
 
-        coffees.forEach(coffee => {
-            let cRoast = coffee.roast.toLowerCase();
-            let cName = coffee.name.toLowerCase();
-            let shouldCoffeeBeAdd = (selectedCoffeeName === "" || cName.indexOf(selectedCoffeeName) >= 0) && (selectedRoast === "all" || cRoast === selectedRoast);
-            if (shouldCoffeeBeAdd === true) {
-                filteredCoffees.push(coffee)
-            }
-        });
-        coffeeContainerElement.innerHTML = renderCoffees(filteredCoffees);
+    let htmlOneOriginalCofffee = `
+    <div class="coffee ${coffee.roast}">
+        <h4 class="w-100 text-center">${coffee.name}</h4>
+        <p class="w-100 text-center">${coffee.roast}</p>
+    </div>
+    `;
 
+    if (coffee.id > 14) {
+        return htmlOneCofffee;
+    } else {
+        return htmlOneOriginalCofffee;
     }
+}
```

Comment:
Original coffee is built without the delete button.  If the coffee list id > 14, it's a new coffee so use the html that adds the delete button, if not use the html that doesn't include the delete button.